python/cugraph/centrality/betweenness_centrality.py

```diff
@@ -11,11 +11,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import random
+import numpy as np
 from cugraph.centrality import betweenness_centrality_wrapper
 
 
+# NOTE: result_type=float could ne an intuitive way to indicate the result type
 def betweenness_centrality(G, k=None, normalized=True,
-                           weight=None, endpoints=False, seed=None):
+                           weight=None, endpoints=False, implementation=None,
+                           seed=None, result_dtype=np.float32):
     """
     Compute the betweenness centrality for all nodes of the graph G. cuGraph
```

Comment:
This could be misinterpreted. I'd recommend :
```suggestion
    Compute the betweenness centrality for all nodes of the graph G from a sample of 'k' sources. cuGraph 
```

Reply:
Updated in 50b27a1